common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -2166,28 +2199,49 @@
   "service.wynntils.itemFilter.sortingNotSupported": "Sorting is not supported for this container",
   "service.wynntils.itemFilter.stat.actual.description": "%s stat of the item",
   "service.wynntils.itemFilter.stat.countedItem.description": "Count of an item (e.g. health potion), this is not the same as the amount of items in a stack.",
+  "service.wynntils.itemFilter.stat.countedItem.name": "Item Count",
```

Comment:
I see you added these name i18ns. Did you add an assert for them in the code?

Reply:
No, the providers for skills and actual stats don't use i18n

Reply:
Oh, okay.